modin/core/execution/ray/implementations/pandas_on_ray/partitioning/__init__.py

```diff
@@ -15,8 +15,16 @@
 
 from .partition import PandasOnRayDataframePartition
 from .partition_manager import PandasOnRayDataframePartitionManager
+from .virtual_partition import (
+    PandasOnRayDataframeVirtualPartition,
+    PandasOnRayDataframeColumnPartition,
+    PandasOnRayDataframeRowPartition,
+)
 
 __all__ = [
     "PandasOnRayDataframePartition",
     "PandasOnRayDataframePartitionManager",
+    "PandasOnRayDataframeVirtualPartition",
+    "PandasOnRayDataframeColumnPartition",
+    "PandasOnRayDataframeRowPartition",
```

Comment:
same as for dask and also not related to the issue

Reply:
https://github.com/modin-project/modin/pull/5456